### PR TITLE
SAF-5198: Add Expiration Interval to Training Wizard

### PR DESCRIPTION
**Pull request by maxswa**

### CODE REVIEW

File: packages/formatron-react/src/form/fluentui/Number.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    setTouched,
+    isTouched,
+    isRequired,
+    isSubmitting
+  } = field;
+
+  return (
+    <TextField
+      value={value ?? ''}
+      type="number"
```
Comment (NickVarnado) [inline packages/formatron-react/src/form/fluentui/Number.tsx:33]: Should this be single quotes?

Comment (maxswa) [reply packages/formatron-react/src/form/fluentui/Number.tsx:33]: `.prettierrc` is missing the `jsxSingleQuote` rule. We can add it as part of this PR if you think it's worth it.

File: packages/safety-suite-api/src/api/models/modelTypes.ts [hunk 2]
```diff
@@ -3361,6 +3472,9 @@ export interface TrainingClassFields {
   description?: string;
   dueDate?: number | null;
   endTime?: number | null;
+  expirationInterval?: number | null;
+  expirationIntervalUnits?: string | null;
+  isExpiring?: string | null;
```
Comment (NickVarnado) [inline packages/safety-suite-api/src/api/models/modelTypes.ts:3477]: Just verifying that `isExpiring` is supposed to be a string and not a boolean.

Comment (maxswa) [reply packages/safety-suite-api/src/api/models/modelTypes.ts:3477]: Yeah it's one of those stupid `Yes` / `No` enums.

File: packages/formatron-react/src/form/fluentui/Enum.tsx [hunk 3]
```diff
@@ -20,16 +22,30 @@ const enumValuesToDropdownOptions = (
...
+  radio?: boolean;
   dropDownProps?: Partial<IDropdownProps>;
 }
 
 export const Enum: React.FC<EnumProps> = ({
   field,
   dropDownProps = {},
-  disabled = false
+  disabled = false,
+  radio = false,
```
Comment (NickVarnado) [inline packages/formatron-react/src/form/fluentui/Enum.tsx:47]: Very nit-picky, but can we follow the boolean naming convention and use `isRadioGroup` here?

File: packages/formatron-react/src/form/fluentui/Number.tsx (hunk 1 above)
Comment (NickVarnado) [reply packages/formatron-react/src/form/fluentui/Number.tsx:33]: eh. I wouldn't worry about it, I was just verifying.

Comment (NickVarnado) [reply packages/formatron-react/src/form/fluentui/Number.tsx:33]: We could add it for consistency if you want, but it's not a showstopper or anything. If we do plan on adding it eventually, then we might as well do it now.

**Review by NickVarnado — APPROVED** [review]

**Pull request merged**